.maintain/rustdocs-release.sh

```diff
@@ -91,6 +91,22 @@ build_rustdocs() {
     && echo "<meta http-equiv=refresh content=0;url=${DOC_INDEX_PAGE}>" > "${2}/index.html"
 }
 
+inject_analytics_script() {
+    local path="$1"
+    local script_content="<script async defer src=\"https://apisa.parity.io/latest.js\"></script><noscript><img src=\"https://apisa.parity.io/latest.js\" alt=\"\" referrerpolicy=\"no-referrer-when-downgrade\" /></noscript>"
+
+    # Define a function that injects our script into the head of an html file.
+    process_file() {
+        local file="$1"
+        echo "Adding Simple Analytics script to $file"
+        sed -i "s|</head>|$script_content</head>|" "$file"
+    }
+    export -f process_file
+
+    # Locate all .html files under the documentation root and use xargs to process in parallel.
+    find "$path" -name '*.html' | xargs -I {} -P "$(nproc)" bash -c 'process_file "$@"' _ {}
```

Comment:
Is `nproc` expected to be installed in this environment? Can you test it in a CI or where is this normally run?